sql/moz-fx-data-shared-prod/telemetry/clients_first_seen/view.sql

```diff
@@ -13,6 +13,12 @@ SELECT
     distribution_id
   ) AS funnel_derived,
   `moz-fx-data-shared-prod`.udf.distribution_model_clients(distribution_id) AS distribution_model,
-  `moz-fx-data-shared-prod`.udf.partner_org_clients(distribution_id) AS partner_org
+  `moz-fx-data-shared-prod`.udf.partner_org_clients(distribution_id) AS partner_org,
+  IF(
```

Comment:
Technically the `IF` here could be removed. Here's an example:
https://github.com/mozilla/bigquery-etl/blob/main/sql_generators/mobile_kpi_support_metrics/templates/active_users.view.sql#L40-L43

```
(
      LOWER(IFNULL(isp, "")) <> "browserstack"
      AND LOWER(IFNULL(distribution_id, "")) <> "
)
```
This would also evaluate to a BOOLEAN value.

Reply:
I'm keeping the original code as that is what DS considers THE UDF to use for this case